web/src/App.tsx

```diff
@@ -117,6 +117,10 @@ export const App: React.FunctionComponent = () => {
                                 setInputHistory={setInputHistory}
                                 isCodyEnabled={true}
                                 onSubmit={onSubmit}
+                                userInfo={{
+                                    isDotComUser: true,
```

Comment:
```suggestion
                                    isDotComUser: isDotCom(config.serverEndpoint),
```
I might be missing some context but wouldn't this not work correctly for enterprise users? Would it be safer to check the serverEndpoint from config to determine its value instead?

Reply:
Used your suggestion 👍